content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,492 @@
+# TYPE probe_http_uncompressed_body_length gauge
+probe_http_uncompressed_body_length 14516
+# HELP probe_http_version Returns the version of HTTP of the probe response
+# TYPE probe_http_version gauge
+probe_http_version 1.1
+# HELP probe_ip_protocol Specifies whether probe ip protocol is IP4 or IP6
+# TYPE probe_ip_protocol gauge
+probe_ip_protocol 4
+# HELP probe_ssl_earliest_cert_expiry Returns earliest SSL cert expiry in unixtime
+# TYPE probe_ssl_earliest_cert_expiry gauge
+probe_ssl_earliest_cert_expiry 1.581897599e+09
+# HELP probe_success Displays whether or not the probe was a success
+# TYPE probe_success gauge
+probe_success 1
+# HELP probe_tls_version_info Contains the TLS version used
+# TYPE probe_tls_version_info gauge
+probe_tls_version_info{version="TLS 1.3"} 1
+```
+
+You can see that the probe was successful and get many useful metrics, like latency by phase, status code, ssl status or certificate expiry in [Unix time](https://en.wikipedia.org/wiki/Unix_time).  
+The blackbox exporter also offers a tiny web interface at [localhost:9115](http://localhost:9115) for you to check out the last few probes, the loaded config and debug information. It even offers a direct link to probe `prometheus.io`. Handy if you are wondering why something does not work.
+
+## Querying multi-target exporters with Prometheus
+
+So far, so good. Congratulate yourself. The blackbox exporter works and you can manually tell it to query a remote target. You are almost there. Now you need to tell Prometheus to do the queries for us.  
+
+Below you find a minimal prometheus config. It is telling Prometheus to scrape the exporter itself as we did [before](#query-exporter) using `curl 'localhost:9115/metrics'`:
+
+NOTE: If you use Docker for Mac or Docker for Windows, you can’t use `localhost:9115` in the last line, but must use `host.docker.internal:9115`. This has to do with the virtual machines used to implement Docker on those operating systems. You should not use this in production.
+
+`prometheus.yml` for Linux:
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - localhost:9115
+```
+
+`prometheus.yml` for macOS and Windows:
+
+```yaml
+scrape_configs:
+
+- job_name: blackbox-exporter-metrics
+  scrape_interval: 5s
+  metrics_path: /metrics
+  static_configs:
+    - targets:
+      - host.docker.internal:9115
+```
+
+Now run a Prometheus container and tell it to mount our config file from above. Because of the way networking on the host is addressable from the container you need to use a slightly different command on Linux than on MacOS and Windows.:
+
+<a name=run-prometheus></a>
+
+Run Prometheus on Linux (don’t use `--network="host"` in production):
+
+```bash
+docker \
+  run --network="host"\
+  --mount type=bind,source="$(pwd)"/prometheus.yml,target=/prometheus.yml,readonly \
```

Comment:
Did you mean $PWD?

Reply:
No, I didn’t.
Please try to avoid passive-aggressive language. It is off-putting and not good for fostering collaboration.

Reply:
Hmm, I didn't realise that this works.

It was an honest question.

Reply:
I am sorry Brian. I read it like a smug "please use this instead". I am very glad you did not mean it like that.
All of this works, I tested it under both, Linux and Mac OS myself and let a new to prometheus engineer try the whole guide to see how it does from a pedagogical perspective as well.

Reply:
Ah no, I'd actually mis-read it as {} rather than () and thus thought that this was a error.